include/pcm_stream_channel_modifier.h

```diff
@@ -8,8 +8,7 @@
 #define _PCM_STREAM_CHANNEL_MODIFIER_H_
 
 #include <zephyr/kernel.h>
```

Comment:
The doxygen defines are missing here. Check the headers of the libraries that you moved previously for reference.

Reply:
(Also, adding them means adding the closing define }@ tags at the end of the header.)